<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-29T20:37:53 IST" name="Default suite" finished-at="2024-02-29T20:38:14 IST" duration-ms="20745">
    <groups>
    </groups>
    <test started-at="2024-02-29T20:37:53 IST" name="Default test" finished-at="2024-02-29T20:38:14 IST" duration-ms="20745">
      <class name="LearnExtentReports">
        <test-method is-config="true" signature="generateReports()[pri:0, instance:LearnExtentReports@4802796d]" started-at="2024-02-29T20:37:53 IST" name="generateReports" finished-at="2024-02-29T20:37:54 IST" duration-ms="1197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReports -->
        <test-method signature="myFirsttest()[pri:0, instance:LearnExtentReports@4802796d]" started-at="2024-02-29T20:37:55 IST" name="myFirsttest" finished-at="2024-02-29T20:38:06 IST" duration-ms="11653" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myFirsttest -->
        <test-method signature="mysecondTest()[pri:0, instance:LearnExtentReports@4802796d]" started-at="2024-02-29T20:38:06 IST" name="mysecondTest" finished-at="2024-02-29T20:38:14 IST" duration-ms="7654" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mysecondTest -->
        <test-method signature="thirdTest()[pri:0, instance:LearnExtentReports@4802796d]" started-at="2024-02-29T20:38:14 IST" name="thirdTest" finished-at="2024-02-29T20:38:14 IST" duration-ms="72" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at LearnExtentReports.thirdTest(LearnExtentReports.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thirdTest -->
      </class> <!-- LearnExtentReports -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
